packages/typespec-python/src/code-model.ts

```diff
@@ -218,25 +218,28 @@ export function emitCodeModel<TServiceOperation extends SdkServiceOperation>(
         clients: [],
         subnamespaceToClients: {},
     };
+    for (const client of sdkPackage.clients) {
+        codeModel["clients"].push(emitClient(sdkContext, client));
+        if (client.nameSpace === sdkPackage.rootNamespace) {
+        } else {
+            codeModel["subnamespaceToClients"][client.nameSpace] = emitClient(sdkContext, client);
+        }
+    }
+    // loop through models and enums since there may be some orphaned models needs to be generated
     for (const model of sdkPackage.models) {
         if (model.name === "" || (model.usage & UsageFlags.Spread) > 0) {
             continue;
         }
-        getType(sdkContext, model);
+        if (!disableGenerationMap.has(model)) {
```

Comment:
since you have changed to generate types alongside with generating clients and operations, i believe this part is unnecessary.

Reply:
see https://github.com/Azure/autorest.python/pull/2740/files#r1719494975. Although we don't generate these models alongside with clients and operations, `sdkPackage.models` still contain these models.